webapp-django/crashstats/urls.py

```diff
@@ -1,5 +1,3 @@
-from __future__ import absolute_import
```

Comment:
Python 2 did this screwy thing with implicit relative imports. I added this line to fix an issue in this file where the implicit relative imports broke things. Python 3 doesn't do implicit relative imports, so we don't need this line anymore.